src/lib/coda_base/ledger.ml

```diff
@@ -189,49 +189,45 @@ struct
     remove_accounts_exn t previous_empty_accounts
 
   (* TODO: Better system needed for making atomic changes. Could use a monad. *)
-  let apply_coinbase t ({proposer; fee_transfer; _} as cb: Coinbase.t) =
+  let apply_coinbase t ({proposer; fee_transfer; amount} as cb: Coinbase.t) =
     let get_or_initialize pk =
       let initial_account = Account.initialize pk in
       match get_or_create_account_exn t pk (Account.initialize pk) with
       | `Added, location -> (location, initial_account, [pk])
       | `Existed, location -> (location, get t location |> Option.value_exn, [])
     in
     let open Or_error.Let_syntax in
-    let%bind proposer_reward, emptys1, receiver_update =
-      match fee_transfer with
-      | None -> return (Protocols.Coda_praos.coinbase_amount, [], None)
-      | Some (receiver, fee) ->
+    let%bind emptys1, receiver_update =
```

Comment:
I think this has to go back to how it was before,  just with `amount` in place of `Protocols.Coda_praos.coinbase_amount`. Right now this is creating more money than it should I think

Reply:
It is, in fact, just replacing `Protocols.Coda_praos.coinbase_amount` with `amount`. Just deleted `proposer_reward` calculation because we don't need it anymore